### ISSUE TITLE
A couple accessibility checks

### ISSUE THREAD

**Issue by KimberlyMunoz** [openoakland/EarlyOakland#26]
Hey all,

I'm working on this for hackforchange. I did a quick accessibility audit and will be submitting a pull request to fix some of these issues.

```
[Severe] ARIA attributes which refer to other elements by ID should refer to elements which exist in the DOM (2)
[Severe] ARIA state and property values must be valid (2)
[Warning] Images should have an alt attribute (1)
See AX_TEXT_02 for more information.
[Warning] Text elements should have a reasonable contrast ratio (18)
See AX_COLOR_01 for more information.
[Warning] The purpose of each link should be clear from the link text (15)
```

![image](https://cloud.githubusercontent.com/assets/1860176/8020913/778fae62-0c59-11e5-9570-d2706f3b7df6.png)

**Comment by spjika** [comment]
Awesome!
